pkg/mutation/assignmetadata.go

```diff
@@ -0,0 +1,72 @@
+package mutation
+
+import (
+	"fmt"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/mutation/path/parser"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// MetadataMutator is a mutator wrapping the AssignMetadata type.
```

Comment:
can we assert that this implements mutator by adding:

```
var _ Mutator = &AssignMetadataMutator{}
```

?

Reply:
Now we can.
Added a test to assert that.